plugin/mmdb/README.md

```diff
@@ -0,0 +1,56 @@
+# mmdb
+## Name
+*mmdb* - Lookup maxmind geoip2 databases then add associated data to the context request.
+
+## Description
+*mmdb* - Allows you to configure [geoIP2 maxmind databases](https://dev.maxmind.com/geoip/docs/databases) and automatically associate and DNS request data matching the client IP.
+
+Common use cases would be adding per request GEO location data, ISP, ASN, etc.
+
+The data will be added to the request context leveraging the [metadata](../metadata/README.md) plugin, and therefore data will be available by make use of it, label names will be created using the following convention.
+
+## Databases
+The following official [max mind databases](https://dev.maxmind.com/geoip/docs/databases) are allowed.
+
+- Anonymous IP
+- ASN
+- City
+- ConnectionType
+- Country
+- Domain
+- Enterprise
+- ISP
+
+Databases with custom schemas are not allowed, but may be supported in the future.
```

Comment:
```suggestion
Databases with custom schemas are not supported yes.
```

Reply:
I guess you meant `s/yes/yet/` - will do.